l10n_it_ddt/readme/USAGE.rst

```diff
@@ -0,0 +1,64 @@
+**English**
+
+You can automatically create a TD From a Sale Order, setting
+'Automatically create the TD' field that will automatically create the TD on
+Sale Order confirmation.
+
+You can also directly create a TD using
+Inventory -> Operations -> TD
+menu and add existings delivery orders to it, in the 'transfers' tab.
+
+You can add lines to an existing TD using the 'Details' tab.
+Lines can be descriptive or linked to a product. If linked to a product,
+the stock movement will also be created.
+
+When you work with delivery orders, you can create a TD selecting 1 or more
+pickings and launching the action 'TD from pickings'.
+
+Also, you can select 1 or more pickings and run 'add pickings to TD' to add
+the selected delivery orders to an existing TD
+
+If the state of the delivery orders allows it, you can deliver them from the
+TD directly, clicking 'put in pack' and 'package done'.
+
+Otherwise, you can process delivery orders separately, then go to the TD and
+click on 'set done'.
+
+Finally you can create your invoice directly from the TD using the
+'Create Invoice' button that creates a new Invoice with the TD lines as
+invoice lines
+
+
+**Italiano**
+
+È possibile creare automaticamente un DDT da un ordine di vendita, impostando
+il campo 'crea automaticamente il DDT' che creerà il DDT alla conferma
+dell'ordine.
+
+È anche possibile creare un DDT direttamente, usando
+Inventario -> Operazioni -> DDT
+e aggiungendo degli ordini di consegna esistenti al DDT, nel tab
+'trasferimenti'.
+
+È possibile aggiungere righe ad un DDT esistente usando il tab 'Dettaglio'.
+Le righe possono essere descrittive o collegate a prodotti. Le righe collegate
+ad un prodotto creeranno anche i movimenti di magazzino.
+
+Se si lavora con gli ordini di consegna, è possibile creare un DDT selezionando
+1 o più ordini di consegna ed eseguendo l'azione 'DDT da Picking'.
+
+Inoltre, è possibile selezionare 1 o più ordini di consegna ed eseguire
+'aggiungi Picking al DDT' per aggiungere gli ordini selezionati ad un DDT
+esistente.
+
+Se lo stato degli ordini di consegna lo permette, è possibile consegnarli tutti
+direttamente dal DDT, cliccando sui bottoni 'metti nel pacco' e
+'pacco completato'.
+
```

Comment:
s/cliccando sui bottoni/facendo clic sui pulsanti

Prendendo come riferimento la revisione già effettuata del modulo "delivery" direi:

s/metti nel pacco/Aggiungi a collo

s/pacco completato/Collo pronto oppure completato

P.S.: poi eventualmente aggiornerò le relative traduzioni.

Reply:
> s/metti nel pacco/Aggiungi a collo
s/pacco completato/Collo pronto oppure completato

Nel modulo stock però questa terminologia non è utilizzata.

Reply:
Il problema è che il modulo stock andrebbe completamente revisionato, insieme ai moduli collegati sale_stock, purchase_stock ecc... (delivery è collegato attraverso la dipendenza sale_stock)
Insieme ad account (che adesso è a un buon punto) è uno dei più corposi di Odoo.
È nella "todo list" delle prorità, vedo di creare un gruppo di lavoro con @mymage, @marcelofrare, @andreampiovesana e @ruben-tonetto.

Reply:
> Aggiungi a collo

Allora metto 'Crea collo' visto che con quel pulsante il collo viene sempre creato

Reply:
In effetti hai ragione, la traduzione corretta dell'azione è "Crea collo".
Modifico anche la stringa in delivery.
Grazie!